prow/cmd/deck/static/pr-script.js

```diff
@@ -439,20 +442,34 @@ function createTidePoolLabel(pr, tidePool) {
  * requirements, it will show that the PR needs to resolve labels.
  * @param isMerge {boolean}
  * @param jobStatus {string}
- * @param mergeAbility {number}
+ * @param noQuery {boolean}
+ * @param labelConflict {boolean}
+ * @param mergeConflict {boolean}
+ * @param branchConflict {boolean}
+ * @param milestoneConflict {boolean}
  * @return {Element}
  */
-function createTitleLabel(isMerge, jobStatus, mergeAbility) {
+function createTitleLabel(isMerge, jobStatus, noQuery, labelConflict, mergeConflict, branchConflict, milestoneConflict) {
     const label = document.createElement("SPAN");
     label.classList.add("title-label");
-    if (isMerge) {
-        label.textContent = "Merged";
-        label.classList.add("merging");
-    } else if (mergeAbility === -1) {
+
+    if (noQuery) {
         label.textContent = "Unknown Merge Requirements";
         label.classList.add("unknown");
-    } else if (mergeAbility === 0) {
-        label.textContent = "Needs to Resolve Labels";
+    } else if (isMerge) {
+        label.textContent = "Merged";
+        label.classList.add("merging");
+    } else if (branchConflict) {
+        label.textContent = "Needs to resolve branch requirements";
```

Comment:
These are misleading, there is nothing for the author to do in these new cases. Maybe phrase like 

    "Blocked from merging into target branch"
    "Blocked from merging by current milestone"

Reply:
That phrasing is a lot clearer. The author *can* change the base branch of the PR, but that would rarely be appropriate.